source/Minable.cpp

```diff
@@ -33,6 +33,9 @@ using namespace std;
 // Load a definition of a minable object.
 void Minable::Load(const DataNode &node)
 {
+	// Set the name of this minable, so we know it has been loaded.
+	name = node.Size() >= 2 ? node.Token(1) : "Unnamed Minable";
```

Comment:
The "Unnamed Minable" bit might not be needed, since minables always have a name. (Instead you could just say, `if(node.Size() >= 2) name = node.Token(1)` to deal with any future case where that's no longer guaranteed.)

Reply:
I had drawn this from Mission::Load, where it gives "Unnamed Mission" if no name is given. But now that I think about how one would go about `using` minables, it is immediately apparent that minables require non-empty names if they are to be used by a system.